### PR TITLE
Add tinuous

### PR DESCRIPTION
**Pull request by jwodder**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

CC: @yarikoptic 

Note to repo maintainers: This package depends on two projects written by me (`in_place` and `click-loglevel`) that are not yet in conda-forge.  PRs for them are located at #14752 and #14751, respectively.

### CODE REVIEW

File: recipes/tinuous/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  summary: Download build logs from GitHub Actions, Travis, and Appveyor
+  description: |
+    tinuous is a command for downloading build logs for a GitHub repository
+    from GitHub Actions, Travis-CI.com, and/or Appveyor.
+  dev_url: https://github.com/con/tinuous
+
+extra:
+  recipe-maintainers:
+    - jwodder
+    - yarikoptic
```
Comment (yarikoptic) [inline recipes/tinuous/meta.yaml:54]: Confirming my willingness to co maintain

**Comment by yarikoptic** [comment]
@jwodder recipe seems to need a bit more work since recipes fail

**Comment by jwodder** [comment]
@yarikoptic The failure appears to be due to not all dependencies being available in conda-forge.  `click-loglevel` has been added, but `in_place` still has not.

**Comment by jwodder** [comment]
The CI is currently failing on Windows only.  The build log ends with:

```
WARNING: failed to get_linkages, codefile_type('C:\Miniconda\conda-bld\tinuous_1622572326321\_h_env\Scripts\tinuous.exe')=EXEfile
   INFO: sysroot: 'C:/Windows/' files: '['winhlp32.exe', 'win.ini', 'twain_32/wiatwain.ds', 'twain_32.dll']'
   INFO (tinuous): Interpreted package 'tinuous' is interpreted by 'python'
[noarch_python] Noarch package contains binary script: tinuous.exe
```

I do not understand why there would be a `tinuous.exe` file involved.

@conda-forge/help-python Help?

File: recipes/tinuous/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 597a43c00c2512b5a82034adc31c36ca84633010f6c186e153b3f438b4ce1271
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (dopplershift) [inline recipes/tinuous/meta.yaml:15]: Add:
```yaml
  entry_points:
    - tinuous = tinuous.__main__:main
```
See [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) for more information.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**